src/components/addTravel/Thumbnail.tsx

```
@@ -27,79 +28,52 @@
-    reader.readAsDataURL(file);
-  };
-
-  const handleFileChange = async (
-    e: ChangeEvent<HTMLInputElement>,
-    setErrMessage: (message: string) => void,
-    onSuccess: (imageData: string) => void,
-  ) => {
+  const handleFileChange = async (e: ChangeEvent<HTMLInputElement>, type: ThumbnailType) => {
     if (e.target.files && e.target.files[0]) {
       const file = e.target.files[0];
       const errorMessage = validateFile(file);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)